src/javascript/app/base/header.js

```diff
@@ -169,6 +175,15 @@ const Header = (() => {
                 on_mobile: true,
 
             },
+            ...(should_show_xtrade ? {
+                derivx: {
+                    name     : 'Deriv X',
+                    desc     : 'Trade FX and CFDs on a customisable, easy-to-use trading platform.',
```

Comment:
Don't we need translation for them? (these and other platforms)

Reply:
we didn't have it before, let me check and add a separate card for it if necessary

Reply:
yes thank you 👍🏻